assets/icons/file_icons/file_types.json

```diff
@@ -22,6 +22,7 @@
         "doc": "document",
         "docx": "document",
         "eex": "elixir",
+        "erl": "erlang",
```

Comment:
I found the issue with this file while editing with stable Zed is it trying to re-format the JSON file to 2 spaces indent. So I temporarily fix by editing outside the Zed to keep the indent.